test/studies/ml/correctness/COMPOPTS

```diff
@@ -0,0 +1 @@
+-M ../lib --fast
```

Comment:
Yes, reading the images without `--fast` takes a long time.

Reply:
How long? Can we use smaller datasets for correctness? Note that these tests are also very important for Chapel's correctness. Compiling without `--fast` is the default and more common way of compilation, so it is important to exercise that. If it is too arduous to create smaller data sets, you can ignore this. But it may be a quick effort with big gain.